### DISCUSSION TITLE
Add NetBox for documenting Home Server

### DISCUSSION THREAD

**Discussion by TomBayne** [swizzin/swizzin#1062] [Feature Requests]
[NetBox](https://docs.netbox.dev/en/stable/installation/) is a tool used for documenting networks. As many people using Swizzin are running Home Lab type environments, this could be very useful.

**Comment by liaralabs** [comment]
This is unfortunately a bit outside the scope of the project. Even just reviewing the dependencies of the project it would be much more suited to running in a containerized/Docker compose environment than on bare metal
